docs/cli/vela_top_design_document.md

```diff
@@ -0,0 +1,70 @@
+---
+title: vela top design document
```

Comment:
this folder are all generated automatically, you'd better move to a new place.

Reply:
How about here: `/docs/platform-engineers/system-operation/ `
Because I find the docs of `vela ql` and `vela system` are also put here.

Reply:
You can move the design doc here: https://github.com/kubevela/kubevela/tree/master/design/vela-cli

This is not user facing doc.

Reply:
OK, I got it, thanks!